src/stat.cc

```diff
@@ -401,10 +401,13 @@ statObjectsOpenfdFilter(const StoreEntry * e)
     if (e->mem_obj == nullptr)
         return 0;
 
-    if (e->mem_obj->swapout.sio == nullptr)
-        return 0;
+    if (e->mem_obj->swapout.sio)
+        return 1;
+
+    if (e->mem_obj->update.reader || e->mem_obj->update.writer)
+        return 1;
 
-    return 1;
+    return 0;
```

Comment:
```suggestion
    return e->diskWriting() ? 1 : 0;
```

And add the corresponding (inlined) diskWriting() methods to StoreEntry and MemObject.

Alternative names to consider: writingToDisk().

We could call the new method swappingOut(), but I am concerned that some future code may misinterpret that as being specific to swapout.sio while the report filter is looking for a more general meaning (that this PR is using to add header update activity to this cache manager report).

Simpler names like writing() would not work because entries being stored into the memory cache or local working memory are not eligible.